students/TimPauley/session03/python_test.py

```diff
@@ -0,0 +1,28 @@
+#Tim Pauley
+#Assignment 03
+#Jan 22 2019
+
+from basic_operations import add_customer
+from customers_model import Customer
+
+
+def test_add_ok_customer():
+	customer_id = 'W3434'
+	name = 'Tim'
+	last_name = 'Pauley'
+	home_address = '1234 Queen Anne Ave N Seattle, Wa 98019'
+	phone_number = '206-123-4567'
+	email_address = 'too_can@gmail.com'
+	status = True
+	credit_limit = '40,0000'
+	add_customer(
+		customer_id
+		,name
+		,last_name
+		,home_address
+		,email_address
+		,status
+		,credit_limit
+		)	
+		Customer.get(Customer.customer_id=customer_id)
+		assert(test_customer.email_address = email_address)
```

Comment:
I think indentation went awry again. Otherwise this test looks like it should work. Not sure if that is your actual information, but remember, these repos are public. For me, info like this is already available on the web, so it doesn't bother me, but want you to be aware of what you are doing, in case you feel differently.